File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/LoginResultVO.kt
package com.sdkwork.backend

data class LoginResultVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val token: String? = null,
    val refreshToken: String? = null,
    val expiresIn: Int? = null,
    val refreshExpiresIn: Int? = null,
    val userId: Int? = null,
    val user: UserVO? = null
)

File: sdkwork-backend-sdk-typescript/src/types/login-result-vo.ts
import type { UserVO } from './user-vo';

/** è®¤è¯ä»¤çä¿¡æ¯ */
export interface LoginResultVO {
  /** åå»ºæ¶é´ */
  createdAt?: string;
  /** æ´æ°æ¶é´ */
  updatedAt?: string;
  /** è®¿é®ä»¤ç */
  token: string;
  /** å·æ°ä»¤ç */
  refreshToken: string;
  /** è®¿é®ä»¤çè¿ææ¶é´(ç§) */
  expiresIn: number;
  /** å·æ°ä»¤çè¿ææ¶é´(ç§) */
  refreshExpiresIn: number;
  /** ç¨æ·id */
  userId: number;
  /** ç¨æ·ä¿¡æ¯ */
  user?: UserVO;
}
